<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CProgConfigHelpLevelPage</class>
 <widget class="QWizardPage" name="CProgConfigHelpLevelPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/newPrefix/resources/BridgeSceneGG3.bmp</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>310</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Please select your level of familiarity with the game of Bridge.  This determines how much help and advice will be presented automatically during the game.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="beginHelpLevel">
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>I've never played bridge before</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="basicHelpLevel">
         <property name="text">
          <string>I know the basic rules of Bridge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="minimumHelpLevel">
         <property name="text">
          <string>I'm already quite comfortable with Bridge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>310</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>You can also have Easy Bridge generate comments and annotations on its own bids and plays.  That is, it can show you what it's thinking when it plays.  While this slows down gameplay somewhat, it can be a valuable learning tool for beginners.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="enableTracing">
         <property name="text">
          <string>Yes, generate and show computer players' analyses</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
